hoot-core-test/src/test/cpp/hoot/core/ConflatorTest.cpp

```diff
@@ -60,6 +61,8 @@ class ConflatorTest : public CppUnit::TestFixture
 
   void runPbfTest()
   {
+    QDir().mkpath("test-output");
+
```

Comment:
Most of the changes are adding a call to make the directory needed in case it isn't there.  `mkpath` is similar to `mkdir -p` in that it will create all necessary directories in the path specified.